dockerfiles/db/setup.sql

```diff
@@ -58,6 +58,7 @@ create table blog_post(
     blog_id integer not null references blog(id) on delete cascade,
     title varchar(255) not null,
     body varchar(10000),
+    tags TEXT[],
```

Comment:
This column was added to allow live validation beyond just the regression tests for filtering by array columns!